forage-android/src/main/java/com/joinforage/forage/android/core/EnvConfig.kt

```diff
@@ -0,0 +1,98 @@
+package com.joinforage.forage.android.core
+
+import com.joinforage.forage.android.ui.ForageContext
+
+enum class EnvOption {
+    DEV,
+    STAGING,
+    SANDBOX,
+    CERT,
+    PROD
+}
+
+sealed class EnvConfig(
+    val FLAVOR: EnvOption,
+    val btProxyID: String,
+    val btAPIKey: String,
+    val vgsVaultId: String,
+    val vgsVaultType: String,
+    val baseUrl: String,
+    val ldMobileKey: String,
+    val ddClientToken: String
+) {
+    object Dev : EnvConfig(
+        FLAVOR = EnvOption.DEV,
+        btProxyID = "N31FZgKpYZpo3oQ6XiM6M6",
+        btAPIKey = "key_AZfcBuKUsV38PEeYu6ZV8x",
+        vgsVaultId = "tntlqkidhc6",
+        vgsVaultType = "SANDBOXbox",
+        baseUrl = "https://api.dev.joinforage.app/",
+        ldMobileKey = "mob-03e025cb-5b4e-4d97-8685-39a22316d601",
+        ddClientToken = "pubf13cedf24ba2ad50d4b9cb0b0100bd4a",
+    )
+
+    object Staging : EnvConfig(
+        FLAVOR = EnvOption.STAGING,
+        btProxyID = "ScWvAUkp53xz7muae7fW5p",
+        btAPIKey = "key_6B4cvpcDCEeNDYNow9zH7c",
+        vgsVaultId = "tnteykuh975",
+        vgsVaultType = "SANDBOXbox",
+        baseUrl = "https://api.STAGINGing.joinforage.app/",
```

Comment:
typo from refactoring across vgsVaultType and baseUrl for all envs.

Reply:
lmao that's embarrassing - thank gawd for code review!

Reply:
address in [eb1c88b](https://github.com/teamforage/forage-android-sdk/pull/102/commits/eb1c88b6dd6a90abca84fbb47959e992b3878673)

Reply:
ACTUALLY address in [ac2f223](https://github.com/teamforage/forage-android-sdk/pull/102/commits/ac2f2237d7550b346f257b88fc2061f9ab8094c8). Gonna mark as resolved!